azure/functions/decorators/function_app.py

```diff
@@ -1938,6 +1968,75 @@ def decorator():
         return wrap
 
 
+class SettingsApi(DecoratorApi, ABC):
```

Comment:
Add pydocs, examples on usage and structure

Reply:
Added docs